app/views/management/_header.html.erb

```diff
@@ -1,4 +1,22 @@
-<nav class="navbar navbar-dark bg-inverse">
+<div class="c-header">
+  <div class="wrapper">
+    <div>
+      <div class="site-switcher js-site-switcher"></div>
+      <% unless @breadcrumbs.nil? or @breadcrumbs.empty? %>
```

Comment:
here you can use
`@breadcrumbs.blank?` that does the same as those two combined ;)